packages/mui-docs/src/branding/brandingTheme.ts

```diff
@@ -1323,6 +1323,23 @@ export function getThemedComponents(): ThemeOptions {
           },
         },
       },
+      MuiSnackbar: {
```

Comment:
The more we use the `brandingTheme`, the more it feels like we should remove this pattern, and replace it with a wrapper of each Material UI component. The file starts to feel bloated. I almost which Material UI had a CLI to automate this extraction.

Now, I guess that with Pigment CSS, moving from JS to CSS would help with performance, but I don't know. What the reality is. Maybe styled(styled()) is actually smaller than the current pattern.

Reply:
I'd like to discuss this further — I don't see this file as bloated; I like that all major customizations live within the theme file. The `styled()` approach would work, but the biggest downside I can think of is that I'd now need to open several different files instead of scrolling up and down on the same one, which sounds like a big DX impact.

Reply:
I was focused on the UX part of the problem. For the DX, yeah, OK maybe.